### ISSUE TITLE
Test Failures (Sep 30, 2015)

### ISSUE THREAD

**Issue by zoffixznet** [cosimo/perl6-lwp-simple#60]
Using latest rakudo/panda, the tests fail:
```bash
==> Fetching LWP::Simple
==> Building LWP::Simple
==> Testing LWP::Simple
t/000-load-module.t ............. ok

# Failed test 'Hostname contains basic auth info'
# at t/basic-auth.t line 16
# expected: 'ron:Camelia@www.software-path.com'
#      got: 'www.software-path.com'

# Failed test 'Basic auth info extracted correctly: user'
# at t/basic-auth.t line 20
# expected: 'ron'
#      got: (Any)


# expected: 'Camelia'
#      got: (Any)

# Failed test 'Basic auth info extracted correctly: pass'
# at t/basic-auth.t line 21
# expected: 'www.software-path.com'
#      got: (Any)
# Looks like you failed 4 tests of 8
t/basic-auth.t .................. 
Dubious, test returned 4 (wstat 1024, 0x400)
Failed 4/8 subtests 
t/custom-headers-and-content.t .. ok
t/get-binary-camelia.t .......... ok
t/get-chunked-6guts.t ........... ok
t/get-perl6-org.t ............... ok
t/get-unsized.t ................. ok
t/get-w3-latin1-utf8.t .......... ok
t/get-w3-redirect.t ............. ok
t/getstore.t .................... ok
t/parse-url.t ................... ok
t/socket-sanity.t ............... ok
t/stringify-headers.t ........... ok

Test Summary Report
-------------------
t/basic-auth.t                (Wstat: 1024 Tests: 8 Failed: 4)
  Failed tests:  2, 5-7
  Non-zero exit status: 4
Files=13, Tests=55, 100 wallclock secs ( 0.09 usr  0.02 sys + 92.53 cusr  2.09 csys = 94.73 CPU)
Result: FAIL
test stage failed for LWP::Simple: Tests failed
```

**Comment by ronaldxs** [comment]
I have a pull request that explains and fixes the problem here: https://github.com/cosimo/perl6-lwp-simple/pull/59.  I explained about it on today irc here: http://irclog.perlgeek.de/perl6/2015-09-30#i_11298393 .

**Comment by zoffixznet** [comment]
This issue has been fixed by #59, merged in https://github.com/cosimo/perl6-lwp-simple/commit/378a558ee9c7149fe118c15e4883cb942a632b7a

All tests now pass! :+1:
